lib/cdo/image_moderation.rb

```diff
@@ -1,15 +1,22 @@
 require 'cdo/azure_content_moderator'
+require 'honeybadger'
 
 module ImageModeration
   # Returns a content rating from an external service.
-  # @param [String] image_url a public url to the image to be rated
+  # @param [IO] image_data - binary image data to be rated
+  # @param [String] content_type - image/bmp, image/gif, image/jpeg, image/png
   # @return [:everyone|:racy|:adult] Whether the image is suitable for everyone
-  def self.rate_image(image_url)
+  def self.rate_image(image_data, content_type)
     return :everyone unless CDO.azure_content_moderation_key
-
     AzureContentModerator.new(
       endpoint: CDO.azure_content_moderation_endpoint,
       api_key: CDO.azure_content_moderation_key
-    ).rate_image(image_url)
+    ).rate_image(image_data, content_type)
+  rescue StandardError => err
```

Comment:
per the comment below, it seems like we should only be swallowing azure errors, so `AzureError` rather than `StandardError`